collector/mssql.go

```diff
@@ -360,7 +361,22 @@ type MSSQLCollector struct {
 	SQLStatsUnsafeAutoParams        *prometheus.Desc
 
 	// Win32_PerfRawData_{instance}_SQLServerSQLErrors
-	SQLErrorsTotal                  *prometheus.Desc
+	SQLErrorsTotal *prometheus.Desc
+
+	// Win32_PerfRawData_{instance}_SQLServerTransactions
+	TransactionsFreeSpaceInTempDbKB           *prometheus.Desc
```

Comment:
Make sure these matches the name of the actual metric we expose, if it is not the same as the WMI name or we improved the naming.